tests/integration/regenerate_template.sh

```diff
@@ -2,31 +2,23 @@
 
 pushd taste
 
+if [[ -n $REGENERATE_BUILD_PROJECT ]]
+then
 echo "INFO: project clean"
 if ! make clean
 then
 	echo "CRITICAL: Cannot clean the project."
 	exit 1
 fi
 
-echo "INFO: project build"
-if ! make
-then
-	echo "CRITICAL: Cannot build project."
-	exit 1
+	   echo "INFO: project build"
+	   if ! make
+	   then
+		   echo "CRITICAL: Cannot build project."
+		   exit 1
+	   fi
 fi
 
-echo "INFO: datamodel"
-# After build the *_datamodel.asn are saved into code subdirectory
-# This loop copies them into src subdirectory
-for datamodel in $(find -name '*_datamodel.asn')
```

Comment:
Why is this removed?

Reply:
It is not necessary to copy files from code to src. Now Opengeode does this.